### PR TITLE
Add left and right margin to center about container

### PR DESCRIPTION
**Pull request by Liberatys**
This pull request adds left/right margins to the content on the about pages. It centers the page content to make it easier to read on smaller devices.

<details>
<summary>Tablet</summary>

image

</details>

<details>
<summary>Phone</summary>

image

</details>

I opted to use the pure-css classes rather than altering the css.

Addresses #1273

### CODE REVIEW

File: templates/core/about/badges.html [hunk 1]
```diff
@@ -4,50 +4,51 @@
...
-		<tr>
-			<td>
-			Version 0.1.0: <a href="{{ mio_badge | safe }}?version=0.1.0">{{ mio_badge }}?version=0.1.0</a>
-			</td>
-			<td><img src="{{ mio_badge | safe }}?version=0.1.0" alt="mio" /></td>
-		</tr>
-		</tbody>
-	</table>
+	<div class="purge-g">
+		<div class="container pure-u-5-6 about">
```
Comment (jyn514) [inline templates/core/about/badges.html:8]: Could you please avoid re-indenting with tabs?

File: templates/core/about/badges.html [hunk 2]
```diff
@@ -4,50 +4,51 @@
...
-
-		<tr>
-			<td>
-			Version 0.1.0: <a href="{{ mio_badge | safe }}?version=0.1.0">{{ mio_badge }}?version=0.1.0</a>
-			</td>
-			<td><img src="{{ mio_badge | safe }}?version=0.1.0" alt="mio" /></td>
-		</tr>
-		</tbody>
-	</table>
+	<div class="purge-g">
```
Comment (jyn514) [inline templates/core/about/badges.html:7]: Is this class documented somewhere? I don't remember it being a custom docs.rs class.

Comment (Liberatys) [reply templates/core/about/badges.html:7]: It's a pure-css grid class: https://purecss.io/grids/

Comment (jyn514) [reply templates/core/about/badges.html:7]: I think you meant to add `pure-g` then, `purge-g` doesn't have any results. Which makes me think this class isn't actually doing anything - does the page look any different if you remove it?

Comment (Liberatys) [reply templates/core/about/badges.html:7]: 🤦 reading is very difficult ^^ the only thing needed is the div.

**Comment by jyn514** [comment]
@Liberatys this looks good to me once you revert the whitespace changes :)

**Pull request merged**